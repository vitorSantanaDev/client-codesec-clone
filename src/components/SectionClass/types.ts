import { ISectionClassProps } from 'interfaces/sections'

export type IViewProps = ISectionClassProps
