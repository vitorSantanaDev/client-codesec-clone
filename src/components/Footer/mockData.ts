export const mockDataFooterProps = {
  logo: {
    altText: 'logo codesec',
    urlImage: '/images/logo-2-codesec.svg'
  },
  footerContent: 'Todos od direitos reservados',
  links: [
    {
      href: '/',
      icon: {
        altText: 'icon social',
        urlImage: '/images/instagram.svg'
      }
    },
    {
      href: '/',
      icon: {
        altText: 'icon social',
        urlImage: '/images/facebook.svg'
      }
    },
    {
      href: '/',
      icon: {
        altText: 'icon social',
        urlImage: '/images/youtube.svg'
      }
    }
  ]
}
