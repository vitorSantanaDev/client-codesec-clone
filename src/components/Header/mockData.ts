import { IViewProps } from './type'

export const mockHeaderProps: IViewProps = {
  logo: {
    urlImage: '/images/codesec-logo.svg',
    altText: 'Logo codesec'
  },
  button: {
    url: '/',
    buttonText: 'Inscreva-se'
  }
}
