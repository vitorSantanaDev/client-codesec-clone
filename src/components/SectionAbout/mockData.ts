export const mockSectionAboutProps = {
  subtitle: 'infos',
  title: 'About web series',
  description:
    'Aprenda PHP do Jeito Certo Utilizando 100% de Tecnologia Moderna e os Melhores Padr√µes de Projeto Nessa Webserie 100% Online e GRATUITA!',
  topics: [
    {
      _id: String(Math.random() * 1000),
      textInfo: 'text 1 example',
      icon: {
        altText: 'icone',
        urlImage: '/images/icon-eye.svg'
      }
    },
    {
      _id: String(Math.random() * 1000),
      textInfo: 'text 1 example',
      icon: {
        altText: 'icone',
        urlImage: '/images/icon-eye.svg'
      }
    },
    {
      _id: String(Math.random() * 1000),
      textInfo: 'text 1 example',
      icon: {
        altText: 'icone',
        urlImage: '/images/icon-eye.svg'
      }
    },
    {
      _id: String(Math.random() * 1000),
      textInfo: 'text 1 example',
      icon: {
        altText: 'icone',
        urlImage: '/images/icon-eye.svg'
      }
    }
  ],
  illustration: {
    urlImage: '/images/about-ilustration.svg',
    altText: 'Illustration about'
  }
}
