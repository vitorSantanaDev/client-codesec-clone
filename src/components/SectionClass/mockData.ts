export const mockDataSectionClassProps = {
  title: 'O que você vai aprender',
  subtitle: 'Aulas',
  classs: [
    {
      dateClass: '14/03',
      hourAndDay: 'Terça as - 20:30H',
      classNumber: 'Aula 01',
      icon: {
        url: 'https://res.cloudinary.com/vsantana/image/upload/v1650500029/icon_lock_6d7fca3f3c.svg',
        altTextIcon: 'Icon'
      },
      titleClass: 'Hackeando tudo',
      descriptionClass:
        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Est mus nisi, sed in tempor. Nisl sit iaculis nulla consectetur. Purus egestas sit eget viverra. Nisi id in porta nec, a a. Eu in dis et, morbi ut at pellentesque. Massa eu amet morbi eu diam ultricies tellus aliquet. Vel, amet dolor cursus lorem quis amet sed. Mollis diam a purus nulla non, mattis proin odio. Facilisis molestie nibh egestas lacus adipiscing.'
    },
    {
      dateClass: '14/03',
      hourAndDay: 'Terça as - 20:30H',
      classNumber: 'Aula 01',
      icon: {
        url: 'https://res.cloudinary.com/vsantana/image/upload/v1650500029/icon_lock_6d7fca3f3c.svg',
        altTextIcon: 'Icon'
      },
      titleClass: 'Hackeando tudo',
      descriptionClass:
        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Est mus nisi, sed in tempor. Nisl sit iaculis nulla consectetur. Purus egestas sit eget viverra. Nisi id in porta nec, a a. Eu in dis et, morbi ut at pellentesque. Massa eu amet morbi eu diam ultricies tellus aliquet. Vel, amet dolor cursus lorem quis amet sed. Mollis diam a purus nulla non, mattis proin odio. Facilisis molestie nibh egestas lacus adipiscing.'
    },
    {
      dateClass: '14/03',
      hourAndDay: 'Terça as - 20:30H',
      classNumber: 'Aula 01',
      icon: {
        url: 'https://res.cloudinary.com/vsantana/image/upload/v1650500029/icon_lock_6d7fca3f3c.svg',
        altTextIcon: 'Icon'
      },
      titleClass: 'Hackeando tudo',
      descriptionClass:
        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Est mus nisi, sed in tempor. Nisl sit iaculis nulla consectetur. Purus egestas sit eget viverra. Nisi id in porta nec, a a. Eu in dis et, morbi ut at pellentesque. Massa eu amet morbi eu diam ultricies tellus aliquet. Vel, amet dolor cursus lorem quis amet sed. Mollis diam a purus nulla non, mattis proin odio. Facilisis molestie nibh egestas lacus adipiscing.'
    },
    {
      dateClass: '14/03',
      hourAndDay: 'Terça as - 20:30H',
      classNumber: 'Aula 01',
      icon: {
        url: 'https://res.cloudinary.com/vsantana/image/upload/v1650500029/icon_lock_6d7fca3f3c.svg',
        altTextIcon: 'Icon'
      },
      titleClass: 'Hackeando tudo',
      descriptionClass:
        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Est mus nisi, sed in tempor. Nisl sit iaculis nulla consectetur. Purus egestas sit eget viverra. Nisi id in porta nec, a a. Eu in dis et, morbi ut at pellentesque. Massa eu amet morbi eu diam ultricies tellus aliquet. Vel, amet dolor cursus lorem quis amet sed. Mollis diam a purus nulla non, mattis proin odio. Facilisis molestie nibh egestas lacus adipiscing.'
    },
    {
      dateClass: '14/03',
      hourAndDay: 'Terça as - 20:30H',
      classNumber: 'Aula 01',
      icon: {
        url: 'https://res.cloudinary.com/vsantana/image/upload/v1650500029/icon_lock_6d7fca3f3c.svg',
        altTextIcon: 'Icon'
      },
      titleClass: 'Hackeando tudo',
      descriptionClass:
        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Est mus nisi, sed in tempor. Nisl sit iaculis nulla consectetur. Purus egestas sit eget viverra. Nisi id in porta nec, a a. Eu in dis et, morbi ut at pellentesque. Massa eu amet morbi eu diam ultricies tellus aliquet. Vel, amet dolor cursus lorem quis amet sed. Mollis diam a purus nulla non, mattis proin odio. Facilisis molestie nibh egestas lacus adipiscing.'
    }
  ]
}
