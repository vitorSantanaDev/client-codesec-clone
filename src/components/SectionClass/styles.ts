import styled, { css } from 'styled-components'
import { Wrapper as Subtitle } from 'components/Subtitle/styles'
import { Wrapper as Heading } from 'components/Heading/styles'
import { Wrapper as Button } from 'components/Button/styles'

export const Wrapper = styled.div`
  ${() => css``}
`

export const HeadingAndSubtitle = styled.div`
  ${({ theme }) => css`
    display: flex;
    align-items: center;
    flex-direction: column;
    margin-bottom: ${theme.spacings.xhuge};

    > ${Subtitle} {
      margin-bottom: calc(${theme.spacings.small} - 0.8rem);
    }

    @media ${theme.media.lteMedium} {
      > ${Heading} {
        text-align: center;
        font-size: calc(${theme.font.sizes.large} + 0.4rem);
      }
    }
  `}
`

export const ContentWrapper = styled.div`
  ${({ theme }) => css`
    width: 100%;
    display: flex;
    flex-wrap: wrap;
    align-items: flex-start;
    justify-content: space-between;
    gap: ${theme.spacings.xxlarge};
    margin-bottom: ${theme.spacings.xlarge};
  `}
`

export const DateWrapper = styled.div`
  ${({ theme }) => css`
    > ${Subtitle} {
      font-size: calc(${theme.font.sizes.small} - 0.1rem);
    }
  `}
`

export const Icon = styled.img`
  ${() => css`
    max-width: 64px;
  `}
`

export const BoxContent = styled.div`
  ${({ theme }) => css`
    position: relative;
    background: ${theme.colors.blackDark};
    padding: ${theme.spacings.medium} ${theme.spacings.xlarge};

    transition: 300ms ease-in-out;
    border-radius: 3px;

    &::before {
      content: '';
      position: absolute;
      inset: 0;
      z-index: -1;
      margin: -1.5px;
      border-radius: 3px;
      background: ${theme.colors.redGradientDark};
    }

    &:hover {
      background: ${theme.colors.redGradientLight};
    }

    > ${Heading} {
      margin-bottom: ${theme.spacings.small};

      @media ${theme.media.lteMedium} {
        font-size: 3rem;
      }
    }
  `}
`

export const ClassNumber = styled.span`
  ${({ theme }) => css`
    display: inline-block;
    font-weight: bold;
    font-size: ${theme.font.sizes.small};
    padding: 0.8rem ${theme.spacings.small};
    margin-bottom: ${theme.spacings.small};

    border-radius: 2px;
    color: ${theme.colors.black};
    background: ${theme.colors.redGradient};
  `}
`

export const Description = styled.p`
  ${({ theme }) => css`
    width: 100%;
    max-width: 537px;

    line-height: 150%;
    font-size: ${theme.font.sizes.small};
    color: ${theme.colors.gray7};
  `}
`

export const ButtonWrapper = styled.div`
  ${({ theme }) => css`
    display: flex;
    justify-content: center;

    > ${Button} {
      margin-top: ${theme.spacings.medium};
    }
  `}
`
