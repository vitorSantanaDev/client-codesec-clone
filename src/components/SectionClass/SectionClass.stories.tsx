import { ComponentStory, ComponentMeta } from '@storybook/react'
import SectionClass from '.'
import { mockDataSectionClassProps } from './mockData'

export default {
  title: 'SectionClass',
  component: SectionClass,
  args: {
    ...mockDataSectionClassProps
  }
} as ComponentMeta<typeof SectionClass>

export const Default: ComponentStory<typeof SectionClass> = (args) => (
  <SectionClass {...args} />
)
