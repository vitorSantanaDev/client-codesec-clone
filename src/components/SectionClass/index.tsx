import Button from 'components/Button'
import Container from 'components/Container'
import Heading from 'components/Heading'
import Subtitle from 'components/Subtitle'
import * as S from './styles'

type SectionClassProps = {
  subtitle: string
  title: string
  classs: {
    dateClass: string
    hourAndDay: string
    classNumber: string
    icon: {
      url: string
      altTextIcon: string
    }
    titleClass: string
    descriptionClass: string
  }[]
}

const SectionClass = ({ subtitle, title, classs }: SectionClassProps) => (
  <Container>
    <S.Wrapper>
      <S.HeadingAndSubtitle data-aos="fade-down">
        <Subtitle>{subtitle}</Subtitle>
        <Heading>{title}</Heading>
      </S.HeadingAndSubtitle>
      {classs?.map((item, index) => {
        const key = `${index}-${item.classNumber}`
        return (
          <S.ContentWrapper key={key}>
            <S.DateWrapper data-aos="fade-right">
              <Heading>{item.dateClass}</Heading>
              <Subtitle>{item.hourAndDay}</Subtitle>
            </S.DateWrapper>
            <S.Icon
              data-aos="fade-down"
              src={item.icon.url}
              alt={item.icon.altTextIcon}
            />
            <S.AnimateWrapper data-aos="fade-right">
              <S.BoxContent>
                <S.ClassNumber>{item.classNumber}</S.ClassNumber>
                <Heading size="medium" as="h4">
                  {item.titleClass}
                </Heading>
                <S.Description>{item.descriptionClass}</S.Description>
              </S.BoxContent>
            </S.AnimateWrapper>
          </S.ContentWrapper>
        )
      })}
      <S.ButtonWrapper data-aos="fade-up">
        <Button size="small">Cadastre-se agora</Button>
      </S.ButtonWrapper>
    </S.Wrapper>
  </Container>
)

export default SectionClass
