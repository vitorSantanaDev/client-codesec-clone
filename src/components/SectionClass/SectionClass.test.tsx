import { screen } from '@testing-library/react'
import { renderTheme } from '../../styles/theme/render-theme'
import SectionClass from '.'
import { mockDataSectionClassProps } from './mockData'

describe('<SectionClass />', () => {
  it('should render section with title and subtitle', () => {
    renderTheme(<SectionClass {...mockDataSectionClassProps} />)
    const heading = screen.getByRole('heading', {
      name: /o que você vai aprender/i
    })
    const subtitle = screen.getByText(/aulas/i)

    expect(heading).toBeInTheDocument()
    expect(subtitle).toBeInTheDocument()
  })
  it('should render correct amount of icons', () => {
    renderTheme(<SectionClass {...mockDataSectionClassProps} />)
    const icons = screen.getAllByAltText(/icon/i)
    expect(icons).toHaveLength(5)
  })
  it('should render the amount of dates and times correctly', () => {
    renderTheme(<SectionClass {...mockDataSectionClassProps} />)
    const datas = screen.getAllByText('14/03')
    const hours = screen.getAllByText('Terça as - 20:30H')
    expect(datas).toHaveLength(5)
    expect(hours).toHaveLength(5)
  })
  it('should render the number of classes correctly', () => {
    renderTheme(<SectionClass {...mockDataSectionClassProps} />)
    const classes = screen.getAllByText(/aula 01/i)
    const classesName = screen.getAllByRole('heading', {
      name: /hackeando tudo/i
    })
    const descriptions = screen.getAllByText(
      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Est mus nisi, sed in tempor. Nisl sit iaculis nulla consectetur. Purus egestas sit eget viverra. Nisi id in porta nec, a a. Eu in dis et, morbi ut at pellentesque. Massa eu amet morbi eu diam ultricies tellus aliquet. Vel, amet dolor cursus lorem quis amet sed. Mollis diam a purus nulla non, mattis proin odio. Facilisis molestie nibh egestas lacus adipiscing.'
    )
    expect(classes).toHaveLength(5)
    expect(classesName).toHaveLength(5)
    expect(descriptions).toHaveLength(5)
  })
  it('should render the call to action button', () => {
    renderTheme(<SectionClass {...mockDataSectionClassProps} />)
    const button = screen.getByRole('button', { name: /cadastre-se agora/i })
    expect(button).toBeInTheDocument()
  })

  it('should match snapshot', () => {
    const { container } = renderTheme(
      <SectionClass {...mockDataSectionClassProps} />
    )
    expect(container.firstChild).toMatchSnapshot()
  })
})
