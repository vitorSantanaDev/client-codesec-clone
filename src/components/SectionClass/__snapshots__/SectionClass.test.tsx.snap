// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionClass /> should match snapshot 1`] = `
.c16 {
  width: 100%;
  max-width: 190px;
  padding: calc(1.6rem - 0.4rem);
  font-weight: 600;
  text-transform: unset;
  font-size: 1.6rem;
  border: none;
  border-radius: 4px;
  cursor: pointer;
  color: #FFFFFF;
  background: linear-gradient(243.46deg,#FA5770 16.66%,#E83752 83.34%);
  -webkit-transition: all ease-in-out 300ms;
  transition: all ease-in-out 300ms;
}

.c16:hover {
  -webkit-transform: scale(1.01);
  -ms-transform: scale(1.01);
  transform: scale(1.01);
  box-shadow: -10px 15px 40px -15px #FA5770;
}

.c0 {
  width: 100%;
  margin: 0 auto;
  max-width: 1200px;
  padding: 3.2rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c6 {
  font-size: 5.6rem;
  color: white;
  font-weight: 600;
  font-family: 'Raleway',sans-serif;
}

.c12 {
  font-size: 2.4rem;
  color: white;
  font-weight: 600;
  font-family: 'Raleway',sans-serif;
}

.c4 {
  background-image: linear-gradient(243.46deg,#FA5770 16.66%,#E83752 83.34%);
  background-clip: text;
  -webkit-background-clip: text;
  -webkit-text-fill-color: transparent;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-weight: 700;
  -webkit-letter-spacing: 15%;
  -moz-letter-spacing: 15%;
  -ms-letter-spacing: 15%;
  letter-spacing: 15%;
  text-transform: uppercase;
  font-size: calc(1.6rem - 0.2rem);
}

.c1 {
  padding: 5.6rem 0;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  margin-bottom: 6.4rem;
}

.c2 > .c3 {
  margin-bottom: calc(1.6rem - 0.8rem);
}

.c7 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  gap: 4.8rem;
  margin-bottom: 4.0rem;
}

.c8 > .c3 {
  font-size: calc(1.6rem - 0.1rem);
}

.c9 {
  max-width: 64px;
}

.c10 {
  position: relative;
  background: #0A0909;
  padding: 2.4rem 4.0rem;
  -webkit-transition: 300ms ease-in-out;
  transition: 300ms ease-in-out;
  border-radius: 3px;
}

.c10::before {
  content: '';
  position: absolute;
  inset: 0;
  z-index: -1;
  margin: -1.5px;
  border-radius: 3px;
  background: linear-gradient(225.05deg,#FA5770 0%,rgba(232,55,82,0) 100%);
}

.c10:hover {
  background: linear-gradient(225.05deg,rgba(250,87,112,0.1) 0%,rgba(232,55,82,0) 50%);
}

.c10 > .c5 {
  margin-bottom: 1.6rem;
}

.c11 {
  display: inline-block;
  font-weight: bold;
  font-size: 1.6rem;
  padding: 0.8rem 1.6rem;
  margin-bottom: 1.6rem;
  border-radius: 2px;
  color: #121111;
  background: linear-gradient(243.46deg,#FA5770 16.66%,#E83752 83.34%);
}

.c13 {
  width: 100%;
  max-width: 537px;
  line-height: 150%;
  font-size: 1.6rem;
  color: #A3A3A3;
}

.c14 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c14 > .c15 {
  margin-top: 2.4rem;
}

@media (max-width:768px) {
  .c6 {
    font-size: 4.0rem;
  }
}

@media (max-width:768px) {
  .c12 {
    font-size: 4.0rem;
  }
}

@media (max-width:768px) {
  .c2 > .c5 {
    text-align: center;
    font-size: calc(3.2rem + 0.4rem);
  }
}

@media (max-width:768px) {
  .c10 > .c5 {
    font-size: 3rem;
  }
}

<div
  class="c0"
>
  <section
    class="c1"
  >
    <div
      class="c2"
      data-aos="fade-down"
    >
      <span
        class="c3 c4"
        color="redGradient"
      >
        Aulas
      </span>
      <h1
        class="c5 c6"
        color="white"
      >
        O que você vai aprender
      </h1>
    </div>
    <div
      class="c7"
    >
      <div
        class="c8"
        data-aos="fade-right"
      >
        <h1
          class="c5 c6"
          color="white"
        >
          14/03
        </h1>
        <span
          class="c3 c4"
          color="redGradient"
        >
          Terça as - 20:30H
        </span>
      </div>
      <img
        alt="Icon"
        class="c9"
        data-aos="fade-down"
        src="https://res.cloudinary.com/vsantana/image/upload/v1650500029/icon_lock_6d7fca3f3c.svg"
      />
      <div
        class=""
        data-aos="fade-right"
      >
        <div
          class="c10"
        >
          <span
            class="c11"
          >
            Aula 01
          </span>
          <h4
            class="c5 c12"
            color="white"
          >
            Hackeando tudo
          </h4>
          <p
            class="c13"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Est mus nisi, sed in tempor. Nisl sit iaculis nulla consectetur. Purus egestas sit eget viverra. Nisi id in porta nec, a a. Eu in dis et, morbi ut at pellentesque. Massa eu amet morbi eu diam ultricies tellus aliquet. Vel, amet dolor cursus lorem quis amet sed. Mollis diam a purus nulla non, mattis proin odio. Facilisis molestie nibh egestas lacus adipiscing.
          </p>
        </div>
      </div>
    </div>
    <div
      class="c7"
    >
      <div
        class="c8"
        data-aos="fade-right"
      >
        <h1
          class="c5 c6"
          color="white"
        >
          14/03
        </h1>
        <span
          class="c3 c4"
          color="redGradient"
        >
          Terça as - 20:30H
        </span>
      </div>
      <img
        alt="Icon"
        class="c9"
        data-aos="fade-down"
        src="https://res.cloudinary.com/vsantana/image/upload/v1650500029/icon_lock_6d7fca3f3c.svg"
      />
      <div
        class=""
        data-aos="fade-right"
      >
        <div
          class="c10"
        >
          <span
            class="c11"
          >
            Aula 01
          </span>
          <h4
            class="c5 c12"
            color="white"
          >
            Hackeando tudo
          </h4>
          <p
            class="c13"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Est mus nisi, sed in tempor. Nisl sit iaculis nulla consectetur. Purus egestas sit eget viverra. Nisi id in porta nec, a a. Eu in dis et, morbi ut at pellentesque. Massa eu amet morbi eu diam ultricies tellus aliquet. Vel, amet dolor cursus lorem quis amet sed. Mollis diam a purus nulla non, mattis proin odio. Facilisis molestie nibh egestas lacus adipiscing.
          </p>
        </div>
      </div>
    </div>
    <div
      class="c7"
    >
      <div
        class="c8"
        data-aos="fade-right"
      >
        <h1
          class="c5 c6"
          color="white"
        >
          14/03
        </h1>
        <span
          class="c3 c4"
          color="redGradient"
        >
          Terça as - 20:30H
        </span>
      </div>
      <img
        alt="Icon"
        class="c9"
        data-aos="fade-down"
        src="https://res.cloudinary.com/vsantana/image/upload/v1650500029/icon_lock_6d7fca3f3c.svg"
      />
      <div
        class=""
        data-aos="fade-right"
      >
        <div
          class="c10"
        >
          <span
            class="c11"
          >
            Aula 01
          </span>
          <h4
            class="c5 c12"
            color="white"
          >
            Hackeando tudo
          </h4>
          <p
            class="c13"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Est mus nisi, sed in tempor. Nisl sit iaculis nulla consectetur. Purus egestas sit eget viverra. Nisi id in porta nec, a a. Eu in dis et, morbi ut at pellentesque. Massa eu amet morbi eu diam ultricies tellus aliquet. Vel, amet dolor cursus lorem quis amet sed. Mollis diam a purus nulla non, mattis proin odio. Facilisis molestie nibh egestas lacus adipiscing.
          </p>
        </div>
      </div>
    </div>
    <div
      class="c7"
    >
      <div
        class="c8"
        data-aos="fade-right"
      >
        <h1
          class="c5 c6"
          color="white"
        >
          14/03
        </h1>
        <span
          class="c3 c4"
          color="redGradient"
        >
          Terça as - 20:30H
        </span>
      </div>
      <img
        alt="Icon"
        class="c9"
        data-aos="fade-down"
        src="https://res.cloudinary.com/vsantana/image/upload/v1650500029/icon_lock_6d7fca3f3c.svg"
      />
      <div
        class=""
        data-aos="fade-right"
      >
        <div
          class="c10"
        >
          <span
            class="c11"
          >
            Aula 01
          </span>
          <h4
            class="c5 c12"
            color="white"
          >
            Hackeando tudo
          </h4>
          <p
            class="c13"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Est mus nisi, sed in tempor. Nisl sit iaculis nulla consectetur. Purus egestas sit eget viverra. Nisi id in porta nec, a a. Eu in dis et, morbi ut at pellentesque. Massa eu amet morbi eu diam ultricies tellus aliquet. Vel, amet dolor cursus lorem quis amet sed. Mollis diam a purus nulla non, mattis proin odio. Facilisis molestie nibh egestas lacus adipiscing.
          </p>
        </div>
      </div>
    </div>
    <div
      class="c7"
    >
      <div
        class="c8"
        data-aos="fade-right"
      >
        <h1
          class="c5 c6"
          color="white"
        >
          14/03
        </h1>
        <span
          class="c3 c4"
          color="redGradient"
        >
          Terça as - 20:30H
        </span>
      </div>
      <img
        alt="Icon"
        class="c9"
        data-aos="fade-down"
        src="https://res.cloudinary.com/vsantana/image/upload/v1650500029/icon_lock_6d7fca3f3c.svg"
      />
      <div
        class=""
        data-aos="fade-right"
      >
        <div
          class="c10"
        >
          <span
            class="c11"
          >
            Aula 01
          </span>
          <h4
            class="c5 c12"
            color="white"
          >
            Hackeando tudo
          </h4>
          <p
            class="c13"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Est mus nisi, sed in tempor. Nisl sit iaculis nulla consectetur. Purus egestas sit eget viverra. Nisi id in porta nec, a a. Eu in dis et, morbi ut at pellentesque. Massa eu amet morbi eu diam ultricies tellus aliquet. Vel, amet dolor cursus lorem quis amet sed. Mollis diam a purus nulla non, mattis proin odio. Facilisis molestie nibh egestas lacus adipiscing.
          </p>
        </div>
      </div>
    </div>
    <div
      class="c14"
      data-aos="fade-up"
    >
      <button
        class="c15 c16"
      >
        Cadastre-se agora
      </button>
    </div>
  </section>
</div>
`;
